examples/interactivity/polygons/feature-click.html

```diff
@@ -5,11 +5,11 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta charset="UTF-8">
```

Comment:
I cannot access the examples from http://127.0.0.1:8080/examples/ I guess it's because of the path change

Reply:
You need to enter in http://127.0.0.1:8080/examples/interactivity because they are not added yet in the `examples.json` file

Reply:
I accessed the example directly, but, don't we want to add them in the `json` file?

Reply:
Yes but not now. These are "internal" examples. We need to discuss with @makella how the examples must be and may be add new examples (oriented to the final user) and then add the reviewed examples to the json file.